typespec-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure-tools/typespec-java",
-  "version": "0.13.9",
+  "version": "0.14.1",
```

Comment:
0.14.0 ?

Also do a npm install, the package-lock.json also need to be updated.